api/app/signals/apps/signals/migrations/0132_migrate_users_signaluser_data.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.13 on 2020-12-14 08:27
```

Comment:
Can you make just 1 migration file? Combining the both into 1 please, we already have way too many migrations files and an item on our backlog to cleanup these files where possible.

Reply:
merged